mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+from mteb.abstasks.AbsTaskClassification import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class ItalianLinguisticAcceptabilityClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="Itacola",
+        dataset={
+            "path": "gsarti/itacola",
+            "revision": "f8f98e5c4d3059cf1a00c8eb3d70aa271423f636",
+        },
+        description="An Italian Corpus of Linguistic Acceptability taken from linguistic literature with a binary annotation made by the original authors themselves.",
+        reference="https://aclanthology.org/2021.findings-emnlp.250/",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["ita-Latn"],
+        main_score="accuracy",
+        date=None,
+        form=["written"],
+        domains=["Non-fiction", "Spoken"],
+        dialect=None,
+        task_subtypes=["Linguistic acceptability"],
+        license=None,
+        socioeconomic_status=None,
```

Comment:
```suggestion
        socioeconomic_status=None, 
```
don't we know this? Where is the data derived from?

Reply:
The data is derived from linguistics textbooks, should that be marked as "high"?

Reply:
I would say so.